app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -238,9 +238,10 @@ def pagination_scope(scope, search_params)
 
       def ordering_scope(scope, ordering_params)
         sort_by = ordering_params[:sort_by] || 'started_at'
+        sort_by = 'foreman_tasks_tasks.' + sort_by if sort_by == 'started_at'
```

Comment:
this raises a question if other columns aren't going to be ambigous :(

Reply:
as of today the sort_by is possible only by started_at and duration. We can improve in future by prefixing everything except for duration quite easily if the need be, so I'm fine taking this as is.